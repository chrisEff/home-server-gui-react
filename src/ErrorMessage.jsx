module.exports = ({message}) =>
	<div style={{color: 'red'}}>{message}</div>
