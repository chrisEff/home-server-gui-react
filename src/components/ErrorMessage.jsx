'use strict'

import React from 'react'
import {StyleSheet, css} from 'aphrodite'

const ErrorMessage = ({message}) =>
	<div className={css([styles.errorMessage])}>{message}</div>

const styles = StyleSheet.create({
	errorMessage: {
		color: 'red',
	},
})

export default ErrorMessage
