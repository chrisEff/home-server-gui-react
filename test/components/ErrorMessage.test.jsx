'use strict'

import React from 'react'
import ErrorMessage from '@/components/ErrorMessage'

import {shallow} from 'enzyme'

describe('ErrorMessage', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<ErrorMessage message='test' />)
		expect(wrapper).toMatchSnapshot()
	})
})
