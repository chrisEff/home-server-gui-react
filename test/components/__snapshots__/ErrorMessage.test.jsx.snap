// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render correctly 1`] = `
<div
  style={
    Object {
      "color": "red",
    }
  }
>
  test
</div>
`;
